src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -110,6 +110,44 @@ auto logical_partial_derivatives(const Variables<VariableTags>& u,
     -> std::array<Variables<DerivativeTags>, Dim>;
 // @}
 
+// @{
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Computes the logical partial derivative of a tensor, prepending the
+ * spatial derivative index, e.g. for \f$\partial_i T_{a}{}^{b}\f$ the C++ call
+ * is `get(i, a, b)`.
+ *
+ * If you have a `Variables` with several tensors you need to differentiate you
+ * should use the `logical_partial_derivatives` function that operates on
+ * `Variables` since that'll be more efficient.
+ */
+template <typename SymmList, typename IndexList, size_t Dim>
+void logical_partial_derivative(
+    gsl::not_null<TensorMetafunctions::prepend_spatial_index<
+        Tensor<DataVector, SymmList, IndexList>, Dim, UpLo::Lo,
+        Frame::Logical>*>
+        logical_derivative_of_u,
+    gsl::not_null<gsl::span<double>*> buffer,
```

Comment:
Maybe in the documentation say why there is a call with and without `buffer` as a parameter? Or just a pointer to the place in the docs where this is explained, if it's explained elsewhere?